### PR TITLE
GH-1612: TDB2 opt file: allow in container directory

### LINKED ISSUE #1612: TDB2 opt file: allow in container directory
### Version

4.6.1

### Feature

Like #1602 (StoreParams), allow the reorder "opt" file to be in the container directory.

For compatibility, a file in a storage directory takes precedence but in the container directory is preferred.

### Are you interested in contributing a solution yourself?

Yes

### PR DESCRIPTION
**Pull request by afs**
GitHub issue resolved #1612

Pull request Description:
Allow "opt" files in the container directory.

----

 - [x] Tests are included.
 - [x] Commits have been squashed to remove intermediate development commit messages.
 - [x] Key commit messages start with the issue number (GH-xxxx or JENA-xxxx)

By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the [Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).

### CODE REVIEW

File: jena-tdb1/src/main/java/org/apache/jena/tdb/sys/Names.java [hunk 1]
```diff
@@ -21,64 +21,64 @@
...
-    public static final String primaryIndexTriples      = "SPO" ; 
+    public static final String primaryIndexTriples      = "SPO" ;
     public static final String[] tripleIndexes          = { primaryIndexTriples, "POS", "OSP" } ;
 
-    public static final String primaryIndexQuads        = "GSPO" ; 
+    public static final String primaryIndexQuads        = "GSPO" ;
     public static final String[] quadIndexes            = { primaryIndexQuads, "GPOS", "GOSP", "POSG", "OSPG", "SPOG"} ;
-    
+
     public static final String primaryIndexPrefix       = "GPU" ;
```
Comment (rvesse) [inline jena-tdb1/src/main/java/org/apache/jena/tdb/sys/Names.java:30]: Not part of this PR but `GPU` - is this a typo?

**Review by rvesse — APPROVED** [review]

File: jena-tdb1/src/main/java/org/apache/jena/tdb/sys/Names.java (hunk 1 above)
Comment (afs) [reply jena-tdb1/src/main/java/org/apache/jena/tdb/sys/Names.java:30]: No :-)

GPU = Graph-Prefix-URI.

It is the prefix storage table.
